docs/pages/components/Loader.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Loader
+description: A rotating spinner to indicate a loading state.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/Loader/index.tsx
+---
+
+import { Loader } from '@deque/cauldron-react'
+
+```js
+import { Loader } from '@deque/cauldron-react'
+```
+
+## Example
+
+```jsx example
+<Loader label="Loading..." />
+```
+
+## Props
+
+<ComponentProps
+  children={true}
+  className={true}
+  refType="HTMLDivElement"
+  props={[
+    {
+      name: 'label',
+      type: 'string',
+      description: '  The desired label will be rendered offscreen. If not provided, aria-hidden="true" will be applied to the element.',
```

Comment:
I'm not sure why there would be spaces here...

```suggestion
      description: 'The desired label will be rendered offscreen. If not provided, aria-hidden="true" will be applied to the element.',
```

Reply:
I thought I pushed that fix 🤔